pkgs/dart_model/lib/src/json_buffer/typed_map.dart

```diff
@@ -0,0 +1,353 @@
+// Copyright (c) 2024, the Dart project authors. Please see the AUTHORS file
```

Comment:
I ran out of steam a bit by the time I got to this one admittedly, but I can take a closer look later 🤣